recipes/default.rb

```diff
@@ -2,24 +2,34 @@
 # Cookbook Name:: gor
 # Attributes:: default
 #
-# Author:: Wouter de Vos <wouter@springest.com>
+# Author:: Wouter de Vos <wouter@springest.com>, Peter de Ruijter <edge@springest.com>
 #
 # Copyright 2012-2013, Wouter de Vos
 #
-cookbook_file '/usr/local/bin/gor' do
-  source 'gor-linux-amd64'
-  owner 'root'
-  mode 0744
-  action :create
-  only_if( "uname -m | grep x86_64" )
+
+include_recipe "golang::default"
+
+
+git "~/gor" do
```

Comment:
Why do you use home folder? I think `/opt/gor` or `/usr/local/gor` would be much better.